packages/@featherds/app-rail/src/components/FeatherAppRailNavItem.vue

```diff
@@ -50,14 +50,20 @@ export default defineComponent({
 @import "@featherds/styles/themes/variables";
 @import "@featherds/styles/themes/utils";
 
+:deep(a.feather-list-item.feather-vertical-app-bar-item:visited) {
+  color: var($state-text-color-on-surface-dark);
```

Comment:
for readability, it is better to insert the style in the hierachy already present below.